actions/image_partition_action.go

```diff
@@ -179,6 +184,21 @@ func (p *Partition) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+func appendFeatures(cmdline []string, features []string) []string {
+	if len(features) > 0 {
+		var tmp strings.Builder
+		for _, o := range features {
+			fmt.Fprintf(&tmp, "%s,", o)
+		}
+		options := tmp.String()
+		// Strip final ","
+		options = options[:tmp.Len()-1]
+		cmdline = append(cmdline, "-O", options)
```

Comment:
You can use strings.Join to concatinate the list with ',' as seperator; that probalby cuts away half of the code :)

Reply:
Also can Features not be nil ? I'd expect taht to be the default

Reply:
> You can use strings.Join to concatinate the list with ',' as seperator; that probalby cuts away half of the code :)

Indeed, thanks for the hint!

> Also can Features not be nil ? I'd expect taht to be the default

I'm not sure what you mean here: Features is a member of the `Partition` struct, can it be nil in a valid `Partition` instance ?